src/app/components/core/dashboard/dashboard.component.ts

```diff
@@ -32,6 +32,7 @@ export class DashboardComponent implements OnInit, OnDestroy {
   private timerSubscription: AnonymousSubscription;
 
   public refreshTimer = GRAPH_REFRESH_INTERVAL;
+  public optedTime;
```

Comment:
I meant to change this name to STATS_HISTORY_TIME_FILTER which will be saved in localStorage! not only the const put into utils.  @monika-Dianomic @mshariq-nerd